src/main/java/frc/robot/subsystems/turretSubsystem/TurretSubsystem.java

```diff
@@ -343,40 +350,30 @@ public void setTurretPos(double setpoint) {
     if (!Constants.enabledSubsystems.turretRotationEnabled) return;
 
     turretGoalPositionEntry.log(Units.radiansToDegrees(setpoint));
-    turretMotor.set(
-        turretPIDController.calculate(
-            turretPosition,
-            MathUtil.clamp(
-                setpoint,
-                Math.toRadians(Constants.TurretConstants.TURRET_MIN_ANGLE_DEGREES),
-                Math.toRadians(Constants.TurretConstants.TURRET_MAX_ANGLE_DEGREES))));
+    turretPositionPIDController.setSetpoint(
+        MathUtil.clamp(
+            setpoint,
+            Math.toRadians(Constants.TurretConstants.TURRET_MIN_ANGLE_DEGREES),
+            Math.toRadians(Constants.TurretConstants.TURRET_MAX_ANGLE_DEGREES)));
   }
 
   public void setPitchPos(double setpoint) {
     if (!Constants.enabledSubsystems.turretPitchEnabled) return;
-
+    usingPitchPid = true;
+    pitchPIDController.setSetpoint(
+        MathUtil.clamp(
+            setpoint,
+            Math.toRadians(Constants.TurretConstants.PITCH_MIN_ANGLE_DEGREES),
+            Math.toRadians(Constants.TurretConstants.PITCH_MAX_ANGLE_DEGREES)));
     pitchGoalPositionEntry.log(Math.toDegrees(setpoint));
-    pitchMotor.setVoltage(
-        pitchPIDController.calculate(
-                pitchPosition,
-                MathUtil.clamp(
-                    setpoint,
-                    Math.toRadians(Constants.TurretConstants.PITCH_MIN_ANGLE_DEGREES),
-                    Math.toRadians(Constants.TurretConstants.PITCH_MAX_ANGLE_DEGREES)))
-            + pitchFeedForward.calculate(pitchPosition, 0));
```

Comment:
Why was this changed?  I'm not understanding, we seem to have removed feed forward entirely?

Reply:
Its just setting the setpoint instead of doing both setpoint and new voltage calculation here. The setVoltage is moved down to periodic (with feedforward). Probably helps with trying to do a startEnd() Command instead of needing a run() Command.